### PR TITLE
give the ID of the user after init session

### PR DESCRIPTION
**Pull request by btry**
Give the ID of the user in the repsonse of Init Session.

Helps to avoid calls to getFullSession when need to know our own ID.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

**Comment by btry** [comment]
@trasher : added a line in CHANGELOG.md and updated the text of initSession in apirest.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The present file will list all changes made to the project; according to the
 - Add and answer approvals from timeline
 - Add lightbox with PhotoSwipe to timeline images
 - Ability to copy tasks while merging tickets
+- the API goves the ID of the user who logs in with initSession
```
Comment (trasher) [inline CHANGELOG.md]: goves? :D

File: apirest.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ App(lication) token
         * "Authorization: user_token q56hqkniwot8wntb3z1qarka5atf365taaa2uyjrn"
 
 * **Returns**:
-  * 200 (OK) with the *session_token* string.
+  * 200 (OK) with the *session_token* string and the *ID of the logged in user*.
```
Comment (trasher) [inline apirest.md]: ` and the *ID of the logged in user* (as of 9.5.0)`?

File: CHANGELOG.md (hunk 1 above)
Comment (btry) [reply CHANGELOG.md]: $fingers = 'fat';

**Pull request merged**